packages/main/src/DateTimePicker.ts

```diff
@@ -385,9 +374,6 @@ class DateTimePicker extends DatePicker {
 	_dateTimeSwitchChange(e: CustomEvent) { // Note: fix when SegmentedButton is implemented in TS
```

Comment:
Check _density getter. It's not used anymore and seems redundant.

Reply:
Done.